src/components/ImportQuizzesButton/index.tsx

```diff
@@ -15,10 +15,9 @@ function ImportQuizzesButton({
       {(props) => (
         <Button
           {...props}
-          title="Import Quizzes"
```

Comment:
why title and className removed

Reply:
Not making it only icon on mobile anymore since we have space now